crates/cargo-bsp/src/project_model/cargo_package.rs

```diff
@@ -1,3 +1,7 @@
+//! Stores necessary information that can be obtained about Cargo package with
+//! available and enabled features (relevant for the feature requests
+//! that are not yet added to the BSP).
```

Comment:
Może zmieniłbym na (relevant for the Cargo extension for BSP, which allows toggling the features, not yet added to the BSP documentation)